.github/workflows/python-build.yaml

```diff
@@ -52,7 +52,12 @@ jobs:
     - name: Test with pytest
       run: |
         pip install pytest pytest-cov
-        pytest test
+        pytest --cov-report=xml --cov=openstf test/
```

Comment:
Ik zou `test/unit`  hier gebruiken denk ik, is iets netter wellicht. Wat het precieze commando moet zijn voor SonarCloud weet ik niet (dat heb jij waarschijnlijk al opgezocht) maar voor SonarQube gebruiken we zoiets: `pytest  --junitxml build/results.xml --cov tasks test/unit --cov-report xml:/venv/bin/coverage-reports/coverage-results.xml`

Reply:
Ok, I added it!